PyMca5/tests/tiled/enable_pymca_import.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+"""enable_pymca_import.py
+
+    Enable tests to import PyMca5 modules.
+
+    By default PyMca5 cannot be imported when called from a source directory.
+    Import _this_ module before importing PyMca5 or one of its submodules
+    to bypass that limitation, or override this behavior
+    by setting the environment varriable ALLOW_PYMCA_IMPORT.
+
+    Examples:
+
+    ```python
+    import enable_pymca_import  # noqa: F401
+
+    # These imports will now succeed!
+    import PyMca5
+    from PyMca5.PyMcaCore.DataObject import DataObject
+    ```
+
+    ```shell
+    # These tests will fail to import PyMca5
+    ALLOW_PYMCA_IMPORT=0 python -m pytest
+    ALLOW_PYMCA_IMPORT=false python -m pytest
+    ALLOW_PYMCA_IMPORT=FALSE python -m pytest
+
+    # These tests will successfully import PyMca5
+    ALLOW_PYMCA_IMPORT=1 python -m pytest
+    ALLOW_PYMCA_IMPORT=true python -m pytest
+    ALLOW_PYMCA_IMPORT=TRUE python -m pytest
+    ```
+
+    Tests that use enable_pymca_import must be in a directory
+    that does NOT contain an __init__.py file.
+"""
+
+import os
+
+
```

Comment:
These helpers are defined separately for the test suite. Although identical to the new functions added to the PyMca5, we should not depend upon the system under test for this functionality.

Reply:
Will handle this later in response to https://github.com/NSLS2/pymca/issues/13